packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -64,6 +64,68 @@ abstract class InteractiveInkFeature extends InkFeature {
     _color = value;
     controller.markNeedsPaint();
   }
+
+  /// Draws an ink splash or ink ripple on the passed in [Canvas].
+  ///
+  /// The [transform] argument gives the coordinate conversion from the coordinate
+  /// system of the canvas to the coordinate system of the [referenceBox].
+  ///
+  /// [Center], [Paint] and [radius] are used to paint the cirle with desired origin,
+  /// paint and radius
+  ///
+  /// [TextDirection] is used for calculating path to clip if [customBorder] is not null.
```

Comment:
```suggestion
  /// [textDirection] is used for calculating path to clip if [customBorder] is not null.
```

At first glance, I'm unclear how [textDirection] is used by [customBorder] or what [customBorder] even is. Maybe start off with what [customBorder] is and then elaborate on how [textDirection] is used with [customBorder] here?

Reply:
Would also be a good place to describe the use of [borderRadius] here as well